src/builtins/BuiltinTemporal.cpp

```diff
@@ -707,16 +835,13 @@ void GlobalObject::installTemporal(ExecutionState& state)
 {
     StaticStrings* strings = &state.context()->staticStrings();
 
-    m_temporal = new Object(state);
-    m_temporal->setGlobalIntrinsicObject(state);
-
-    m_temporalNow = new Object(state);
+    auto m_temporalNow = new Object(state);
     m_temporalNow->setGlobalIntrinsicObject(state);
 
-    m_temporalPlainDate = new NativeFunctionObject(state, NativeFunctionInfo(strings->lazyDate(), builtinTemporalPlainDateConstructor, 4), NativeFunctionObject::__ForBuiltinConstructor__);
+    auto m_temporalPlainDate = new NativeFunctionObject(state, NativeFunctionInfo(strings->lazyDate(), builtinTemporalPlainDateConstructor, 4), NativeFunctionObject::__ForBuiltinConstructor__);
     m_temporalPlainDate->setGlobalIntrinsicObject(state);
 
-    m_temporalPlainDatePrototype = new PrototypeObject(state);
+    auto m_temporalPlainDatePrototype = new PrototypeObject(state);
     m_temporalPlainDatePrototype->setGlobalIntrinsicObject(state, true);
```

Comment:
Please remove prefix `m_` for these temporal variables.
The prefix `m_` generally represents member element of class/structure.